### PR TITLE
Whats new blockaid popup

### PR DESCRIPTION
**Pull request by segun**
## Explanation

Let's communicate to users this amazing new feature that we have!! 🎉

Acceptance Criterias

A new message with image + text should be added to our existing What's new modal
Clicking on Enable security alerts should take the user to this feature within Settings (and potentially also turn the feature on - TBD)
This should be behind the same feature/build flag that we have for the rest of Blockaid integration feature

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/927

### Before
oldwn

### After
Screenshot 2023-07-25 at 18 01 32

## Manual Testing Steps

- install MM afresh
- Register/Import wallet
- What's new popup should be shown and blockaid should be second what's new on the popup after swaps what's new
## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by jpuri — DISMISSED** [review]

**Comment by bschorchit** [comment]
On the screenshot above the image within the What's new seems a bit pixelated cc: @SayaGT. Could we have it in better resolution? Or is it pixelated in this screenshot?

File: test/e2e/tests/security-provider.spec.js [hunk 1]
```diff
@@ -1,235 +0,0 @@
...
-const FixtureBuilder = require('../fixture-builder');
-
-const OPENSEA_URL =
-  'https://proxy.metafi.codefi.network/opensea/security/v1/validate';
-
-/**
- * @param {import('mockttp').Mockttp} mockServer - The mock server.
- */
-
-describe('Transaction security provider', function () {
```
Comment (cryptotavares) [inline test/e2e/tests/security-provider.spec.js:12]: We are still supporting this in main.. I don't think that we should remove this tests? Was there any reason for removing them?

File: ui/selectors/selectors.js [hunk 2]
```diff
@@ -1008,6 +1008,9 @@ function getAllowedAnnouncementIds(state) {
     19: false,
     20: currentKeyringIsLedger && isFirefox,
     21: isSwapsChain,
+    ///: BEGIN:ONLY_INCLUDE_IN(blockaid)
+    23: true,
```
Comment (cryptotavares) [inline ui/selectors/selectors.js:1012]: shouldn't it be 22?

**Review by jpuri — APPROVED** [review]
LGTM

File: ui/selectors/selectors.js (hunk 2 above)
Comment (segun) [reply ui/selectors/selectors.js:1012]: There's already a 22 in the extension

```
shared/notifications/index.js
test/e2e/fixture-builder.js
```

But for some reason it wasn't added here.

**Comment by segun** [comment]
> On the screenshot above the image within the What's new seems a bit pixelated cc: @SayaGT. Could we have it in better resolution? Or is it pixelated in this screenshot?

@bschorchit I downloaded the image directly from figma, here's the original...I think it's pixelated in the SS

blockaid-security-provider

**Review by cryptotavares — APPROVED** [review]

**Pull request merged**